eventmesh-webhook/eventmesh-webhook-receive/src/main/java/org/apache/eventmesh/webhook/receive/storage/HookConfigOperationManage.java

```diff
@@ -97,10 +94,10 @@ public WebHookConfig queryWebHookConfigById(WebHookConfig webHookConfig) {
         if(filePattern) return cacheWebHookConfig.get(webHookConfig.getCallbackPath());
         else{
             try {
-                String content = configService.getConfig(webHookConfig.getManufacturerEventName() + DATA_ID_EXTENSION, GROUP_PREFIX + webHookConfig.getManufacturerName(), TIMEOUT_MS);
+                String content = configService.getConfig(MD5Utils.md5Hex(webHookConfig.getCallbackPath(), "UTF_8") + WebHookOperationConstant.DATA_ID_EXTENSION, WebHookOperationConstant.GROUP_PREFIX + webHookConfig.getManufacturerName(), WebHookOperationConstant.TIMEOUT_MS);
```

Comment:
Can we add a utils method to do this convert? If we change the path parten, we need to modified evenywhere.